src/image/ImageAnnotator.ts

```diff
@@ -0,0 +1,87 @@
+import BaseAnnotator from '../common/BaseAnnotator';
+import BaseManager from '../common/BaseManager';
+import { CLASS_ANNOTATIONS_LOADED } from '../constants';
+import { getAnnotation } from '../store/annotations';
+import { centerRegion, isRegion, RegionManager } from '../region';
+import './ImageAnnotator.scss';
+import { scrollToLocation } from '../utils/scroll';
+
+export default class ImageAnnotator extends BaseAnnotator {
+    annotatedEl?: HTMLElement;
+
+    managers: Set<BaseManager> = new Set();
```

Comment:
Do we still need this to be a set of managers? If we end up support multi page images, wouldn't it need to be similar to `DocumentAnnotator` where it's keyed by page?

Reply:
Yeah, it's meant to support multiple types of annotations. The DocumentAnnotator uses a Map (pages) of Sets (annotation types), instead.